docs/index.md

```diff
@@ -1,19 +1,56 @@
-[Islandora](https://islandora.ca) is an open-source framework that provides the necessary tools turn a [Drupal 8](https://www.drupal.org) website into a fully-functional preservation repository for digital assets without the need for additional code, while remaining flexible enough to be extended and customized.
+# Overview
 
-Out of the box, this includes:
+[Islandora](https://islandora.ca) is an open-source framework that provides the necessary tools turn a [Drupal](https://www.drupal.org) website into a fully-functional preservation repository for digital assets without the need for additional code, while remaining flexible enough to be extended and customized.
```

Comment:
I'm not comfortable saying Islandora is a "fully-functional preservation repository for digital assets". It can't replace Archivematica for most users of that system, for example. I think it's more accurate to call Islandora a Digital Assets Management System that has a lot of preservation capabilities.

Reply:
Mind you, the only important thing Islandora can't currently do that Archivematica can is [enact configurable preservation policies](https://www.archivematica.org/en/docs/archivematica-1.11/user-manual/preservation/preservation-planning/#pres-policies). It could do it if we wanted it to.

Reply:
Agreed! I didn't have time to rewrite this entire page, though it definitely needs it!

Reply:
I'm happy to propose wording - should I do that here, or open another PR once this is merged?

Reply:
here's fine! I was thinking less "preservation repository" more "digital asset management system" but the rest of the sentence would need balancing out... what are you thinking?

Reply:
Maybe

> [Islandora](https://islandora.ca) is an open-source framework that provides the necessary tools turn a [Drupal](https://www.drupal.org) website into a fully-functional Digital Assets Management System. 

And then add under the "Islandora:" section a note about its digital preservation capabilities. I've been informally maintaining https://github.com/mjordan/digital_preservation_using_islandora just to keep some of this stuff in one place, but so far, only Fedora and FITS have been mainstreamed into core Islandora. Riprap, Bagger, PREMIS and Reports have not yet undergone LSAP or adoption in to core. So not sure what level of detail to provide in the "Islandora:" section entry. Would be happy to hear what others think.

Reply:
Seems like something that should be explained in the appropriate section of the documentation (and the non-core ones maybe getting more attention with this recent survey)

Reply:
@mjordan that documentation about preservation is awesome, can I propose it be added to the [Cookbook](https://github.com/Islandora-Labs/Islandora-Cookbook)?

Reply:
Sure, that would be great!

Reply:
I'll do a PR for that